cli/pcluster/cfnconfig.py

```diff
@@ -801,6 +800,10 @@ def __init_fsx_parameters(self):
         if not fsx_section:
             return
 
+        # Check that the base_os is supported
+        if self.__config.has_option(self.__cluster_section, "base_os"):
```

Comment:
we should consider also the case where `base_os` is not set but uses default `alinux` value

Reply:
whoops. ok updated now